src/Model/FIT/GlobalProfileAccess.php

```
@@ -0,0 +1,101 @@
+  public static function getFieldTypeValue(string $field_type_name, int $key)
+  {
+    return isset(GlobalProfile::FIELD_TYPES[$field_type_name]['values'][$key])
+      ? GlobalProfile::FIELD_TYPES[$field_type_name]['values'][$key]
+      : 'unknown'
+    ;
+  }
+
+  public static function getAllFieldDefinitions()
+  {
+    foreach (GlobalProfile::MESSAGE_TYPES as $message_global_number => $message_definition) {
+      foreach($message_definition['fields'] as $field_number => $field){
```

PHPMD (2.15.0)

Avoid unused local variables such as '$field'. (Unused Code Rules)

(UnusedLocalVariable)